### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by amklass11**
# add remove functions to Remove  npm run server to start

1. add task
2. remove the task
3. edit task
 All functionalities added

### CODE REVIEW

File: src/modules/htmlElements.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const ul = document.querySelector('body ul');
+const input = document.getElementById('add-input');
```
Comment (OLIPLICHE) [inline src/modules/htmlElements.js:2]: - [ ] Please Make sure a function for adding a new task (**add a new element to the array**) is well implemented and works fine.

File: src/modules/editingTask.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const elementLi = document.querySelector(selectedElementClass);
+  const elementInput = event.currentTarget;
+  elementLi.classList.remove('editing-background');
+  elementInput.classList.remove('editing-background');
+  elementInput.addEventListener('input', modifyTask(elementInput.value, selectedElementClass));
+  const icon = document.querySelector(`${selectedElementClass} > img`);
+  icon.setAttribute('src', './assets/images/dot.png');
+}
+
+function editingTask(event) {
```
Comment (OLIPLICHE) [inline src/modules/editingTask.js:14]: - [ ] Please your app is not working. Try and work through then submit.
-
sss

- [ ] Kindly stick to project requirements ([See Project](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md))

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @amklass11 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is descriptive 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/editingTask.js (hunk 2 above)
Comment (amklass11) [reply src/modules/editingTask.js:14]: @OLIPLICHE  Hello, I tested it and it worked, we needed to run it on a local server. to see it

File: src/modules/editingTask.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    list.forEach((element) => {
+      if (element.index !== Number(p.classList[0].slice(5))) {
+        newlist.push(element);
+      }
+    });
+    let newindex = 0;
+    elementLi.remove();
+    newlist.forEach((element) => {
+      element.index = newindex;
+      newindex += 1;
```
Comment (danielufeli) [inline src/modules/editingTask.js:38]: - [OPTIONAL] Nice work done on your Todo list, kindly set the property index to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). It is currently starting at 0.

**Review by danielufeli — APPROVED** [review]
Hi @amklass11,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**